resources/views/livewire/slack-settings-form.blade.php

```diff
@@ -12,33 +12,33 @@
 {{-- Page content --}}
 @section('content')
 
-<div>
+<div><!-- livewire div - do not remove -->
```

Comment:
For what it's worth, so long as you return one 'root node', LiveWire doesn't care what it is. So you *could* pull this `<div>` and just the `<form>` if you wanted. But it doesn't really matter.